doc/authorization.md

```diff
@@ -0,0 +1,115 @@
+for the National Capital Region (NCR) there are `WorkOrder` records, each of which is
+related to exactly one *Proposal*.
+
+*ClientData* records contain client-specific funding request details: building or vendor
+names, codes or numbers. Each Client may have different validation rules around their
+respective details: required, min/max ranges, etc. A full list of *ClientData* specifics
+is in the database schema document [`db/schema.rb`](../db/schema.rb) and the related Model
+classes.
+
+*ClientData* is not PII and is not considered legally sensitive; however *ClientData* is siloed
+as a UX convenience for Users within the system.
+
+## Roles
+
+A User may be assigned one or more roles in relation to the system as a whole (e.g. Admin).
+System roles are marked with an :anchor:.
+
+A User may be assigned one or more roles in relation to a specific Proposal (e.g. Approver).
+Proposal roles are marked with a :clipboard:.
+
+See also discussion in [the Roles section of Overview doc](overview.md#roles).
+
+### Approver :clipboard:
+
+A User who MAY directly approve a Proposal.
+
+### Client-specific approvers :clipboard:
+
+There are several client-specific approvers, which typically represent mailboxes not tied
+to a particular human being but that instead use Delegates. E.g. `BA61_tier1_budget_approver`
+and `BA61_tier2_budget_approver`.
+
+### Delegate :clipboard:
+
+A User who MAY approve Proposals on behalf of an Approver.
+
+### Observer :clipboard:
+
+A User who MUST receive notifications for and MAY comment on a Proposal.
+
+### Requester :clipboard:
+
+The User who initiated a Proposal.
+
+### Subscriber :clipboard:
+
+An umbrella term, referring to an Approver, Delegate, Observer or Requester.
+I.e. a User explicitly associated with a Proposal.
+
+### Admin :anchor:
+
+A User who MAY act on any Proposal or User in the system. Access to the *Admin* role
+may only be granted by another User with the *Admin* role (via the `/admin` UI) or by
+someone with console access (e.g. via `cf-ssh`).
+
+### Client Admin :anchor:
+
+A User who MAY view all Proposals with the same `client_slug` as part of their dashboard
+aggregation reporting. NOTE this aggregation feature is a UX convenience and does not
+reflect any data sensitivity, since any User MAY view any Proposal with a consisten `client_slug`. 
+
+## Rules
+
+When these rules refer to a *Proposal* they mean also the ClientData associated with the Proposal.
+
+1. An Admin User MAY act upon any object in the system.
+
+1. Any User MAY create a new Proposal consistent with their `client_slug`. E.g.
+a User with a `client_slug` of **ncr** MAY only create a NCR `WorkOrder` and not any other
+kind of Proposal.
+
+1. Any Subscriber MAY add another User with the same `client_slug` as an Observer
+to the subscribed Proposal.
+
+1. Only the Requester of a Proposal MAY edit or cancel the Proposal.
+
+1. Any Subscriber MAY comment on a Proposal, which will add that User as
```

Comment:
This might be more specific: "Any subscriber for a Proposal MAY comment on that Proposal," -- not sure if being super specific matters here